storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbInstanceFactory.java

```diff
@@ -164,24 +165,71 @@ private static DBOptions createDBOptions(
   }
 
   private static ColumnFamilyOptions createColumnFamilyOptions(
-      final KvStoreConfiguration configuration, final Cache cache) {
-    return new ColumnFamilyOptions()
-        .setCompressionType(configuration.getCompressionType())
-        .setBottommostCompressionType(configuration.getBottomMostCompressionType())
-        .setLevelCompactionDynamicLevelBytes(true)
-        .setTableFormatConfig(createBlockBasedTableConfig(cache));
+      final KvStoreConfiguration configuration, final KvStoreColumn<?, ?> column) {
+    final ColumnFamilyOptions cfOptions;
+    try {
+      final LRUCache cache =
+          column
+              .getIsLargerCacheAvalilable()
+              .map(
+                  isLarger -> {
+                    if (isLarger) {
+                      LOG.info("Using larger cache for column {}", column.getId().toHexString());
+                      return new LRUCache(configuration.getLargerCacheCapacity());
+                    } else {
+                      return new LRUCache(configuration.getCacheCapacity());
+                    }
+                  })
+              .orElse(new LRUCache(configuration.getCacheCapacity()));
```

Comment:
If `getIsLargerCacheAvalilable()` is Optional.empty(), it means that we are on the default case, right? If we don't want to change the existing behaviour we should use the large capacity (matching the old value) when getIsLargerCacheAvalilable() is empty. Maybe I am missing something but it does look like to me that we would be changing every cache to 32MB.

Reply:
So long story short the default value used to be 8MB, which is low. We've changed to 128MB as a first step to get the db performing better and we're trying to narrow down what columns need larger caches, that being said the default should be something lower than 128MB, Rocks default is 32MB (as of release 8.x) which is what I believe we should be using by default as well, 8MB was set long time ago and I don't think is suitable anymore for the current needs.